### ISSUE TITLE
Do not flag CA1056 on virtual method implementation

### ISSUE THREAD

**Issue by 333fred** [dotnet/roslyn-analyzers#3146]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

v2.9.8 (Latest)

#### Diagnostic ID

[CA1056](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1056?view=vs-2019)

#### Repro steps

```cs
using System;
public class Test : I
{
    public string UrlPathSegment // Flagged here
        => throw new NotImplementedException();
}

public interface I
{
    string UrlPathSegment // And here
        { get; }
}
```

I expect to see the second warning, as it's my interface and it's under my control. But the first warning, particularly when the type is coming from a different assembly (such as `ReactiveUI.IRoutableViewModel`) isn't something I can change.
